client/src/js/services/EnterpriseService.js

```diff
@@ -0,0 +1,49 @@
+'use strict';
```

Comment:
On the client, you do not need to put the ECMAScript 5 directive `use strict`.  It is added during the build process.